specification/automation/resource-manager/Microsoft.Automation/preview/2023-05-15-preview/examples/createOrUpdateConnection.json

```diff
@@ -0,0 +1,62 @@
+{
+  "parameters": {
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg",
+    "automationAccountName": "myAutomationAccount28",
+    "connectionName": "mysConnection",
+    "api-version": "2023-05-15-preview",
+    "parameters": {
+      "name": "mysConnection",
+      "properties": {
+        "description": "my description goes here",
+        "connectionType": {
+          "name": "Azure"
+        },
+        "fieldDefinitionValues": {
+          "AutomationCertificateName": "mysCertificateName",
+          "SubscriptionID": "subid"
+        }
+      }
+    }
+  },
+  "responses": {
+    "201": {
+      "headers": {},
+      "body": {
+        "id": "/subscriptions/subid/resourceGroups/rg/providers/Microsoft.Automation/automationAccounts/myAutomationAccount28/connections/mysConnection",
+        "name": "mysConnection",
+        "properties": {
+          "creationTime": "2017-03-27T07:52:29.29+00:00",
+          "lastModifiedTime": "2017-03-27T07:52:29.29+00:00",
+          "description": "my description goes here",
+          "fieldDefinitionValues": {
+            "AutomationCertificateName": "mysCertificateName",
+            "SubscriptionID": "subid"
+          },
+          "connectionType": {
+            "name": "Azure"
+          }
```

Comment:
curious - why doe it look like this instead of just e.g.

   "connectionType": "Azure"
?

Reply:
This ConnectionType is further object under Connection-
specification/automation/resource-manager/Microsoft.Automation/preview/2023-05-15-preview/examples/createOrUpdateConnectionType.json